testflight/set-pipeline-step_test.go

```diff
@@ -51,39 +55,56 @@ jobs:
         greetings: hello
 `
 
-	var fixture string
+	const pipelineContentWithTeam = pipelineContent + `
```

Comment:
nit: This looks kind of brittle; can we do this with a static `((var))` instead? We can run with `-y team=""`, `-y team=testflight`, and `-y team=main`.

Reply:
Hadn't thought of that, yes I will

Also I will move the inline template into a fixture